.changes/next-release/Bug Fix-00926769-d0c3-4810-8f46-46973a0d5e49.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "S3: use getBucketLocation to lookup bucket regions"
```

usually want this to say something that is meaningful to the customer, e.g.:
```suggestion
	"description": "S3: improved performance in private VPC (via getBucketLocation)"
```

---

Gotcha, good call.